### PR TITLE
Fix bundler dependency

### PR DESCRIPTION
**Pull request by abtris**

### CODE REVIEW

File: lib/redsnow/object.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Object
     # Symbolizes keys of a hash
     def deep_symbolize_keys
       return each_with_object({}) { |memo, (k, v)| memo[k.to_sym] = v.deep_symbolize_keys } if self.is_a?(Hash)
-      return each_with_object([]) { |memo, v | memo << v.deep_symbolize_keys } if self.is_a?(Array)
+      return each_with_object([]) { |memo, v| memo << v.deep_symbolize_keys } if self.is_a?(Array)
```
Comment (pksunkara) [inline lib/redsnow/object.rb]: What does this change do?

Comment (pksunkara) [inline lib/redsnow/object.rb]: Sorry. I thought you changed `Hash` to `Array`.

**Comment by jesperronn** [comment]
Could you please tag and release new version so that we can use it in my project.

**Comment by jesperronn** [comment]
@pksunkara, @abtris  can you provide a new release (0.3.6 or similar). Rubygems still only holds 0.3.4. I desperately need this one for a project that gives me strange errors because of the bundler dependency in redsnow.

Error due to 0.3.4 specification from my project:

```
$ bundle update rails
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/..
Resolving dependencies...
Bundler could not find compatible versions for gem "bundler":
  In Gemfile:
    redsnow (>= 0) ruby depends on
      bundler (~> 1.7.0) ruby

  Current Bundler version:
    bundler (1.8.4)

This Gemfile requires a different version of Bundler.
Perhaps you need to update Bundler by running `gem install bundler`?

```

**Pull request merged**